### ISSUE TITLE
[轉載] Styling SVG <use> Content with CSS

### ISSUE THREAD

**Issue by aleen42** [aleen42/PersonalWiki#18]
The article, written by @SaraSoueidan, is talking about how to style contents of `<use>` elements in SVG with CSS. I hope posting here in order to make it more widely useful for Front-end developers. Soon or later, I would translate it into a Chinese version. May I, @SaraSoueidan?
- Read at: https://aleen42.gitbooks.io/personalwiki/content/post/style_svg_use/style_svg_use.html

**Comment by SaraSoueidan** [comment]
@aleen42 Yes, absolutely. Please provide a link to the original article on Codrops and get @crnacura’s approval before you do too.

Thanks!

**Comment by aleen42** [comment]
@SaraSoueidan Okay, I will ask for approval soon or later.

**Comment by aleen42** [comment]
@SaraSoueidan How can I get in touch with @crnacura? Email or directly @ her?

**Comment by aleen42** [comment]
@SaraSoueidan, @crnacura, Hello, here is a [**Chinese version**](https://aleen42.gitbooks.io/personalwiki/content/translation/style_svg_use/style_svg_use.html#).
